Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/FileProcessor/NewIshObfuscatedFile.Tests.ps1

```diff
@@ -34,7 +34,8 @@ try {
 
 Describe “New-IshObfuscatedFile" -Tags "Read" {
 	Write-Host "Initializing Test Data and Variables"
-	$rootFolder = Join-Path -Path $env:TEMP -ChildPath $cmdletName 
+	$tempFolder = [System.IO.Path]::GetTempPath()
```

Comment:
Why a C# statement? Why was `$env:TEMP` not good enough?

Reply:
$env:Temp returns C:\Users\DMYKHA~1\AppData\Local\Temp
c# statement returns C:\Users\dmykhalchenko\AppData\Local\Temp\

Reply:
So both work... the top one is the DOS style naming, the bottom is Windows style. Did it fail for you?

Reply:
Yes, it did. In comparison it returns false for the same file paths.

Reply:
Did it gave an error in the test? And if this is an issue, why did you not change the usage in the finally?

Reply:
Yes it did. This is not issue for Join-Path function which is used in the finally, only for comparison with full name of FileInfo object.

Reply:
Strange the code seems not be changed, but you are right I also get an exception when I try to run that test. The following line seems to fail:
($taskFileInfo.FullName -eq (Join-Path -Path $outputFolder -ChildPath "task==1=en.xml")) | Should Be $True